app/posts/manage-breast-screening/2025/04/2025-04-08-understanding-image-reading.md

```diff
@@ -14,6 +14,8 @@ Image reading (or film reading) is when trained staff check mammograms from brea
 
 It's often called ‘film reading’ because traditionally the process was done with actual x-ray films. Although mammograms are now digital, the term 'film reading' remains in common use.
 
```

Comment:
I know this is super annoying, but this is the first mention of PACs and NBSS. I read further down and then understood the acronyms. So this is a comment that you can ignore - it just struck me a bit